packages/sensors_plus/sensors_plus_platform_interface/lib/sensors_plus_platform_interface.dart

```diff
@@ -102,4 +104,12 @@ abstract class SensorsPlatform extends PlatformInterface {
   }) {
     throw UnimplementedError('magnetometerEvents has not been implemented.');
   }
+
+  /// Returns a broadcast stream of events from the device barometer at the
+  /// given sampling frequency.
+  Stream<BarometerEvent> barometerEventStream({
+    Duration samplingPeriod = SensorInterval.normalInterval,
+  }) {
+    throw UnimplementedError('barometerEvents has not been implemented.');
+  }
```

Comment:
Since we modify the platform interface, this change has to be marked as a breaking change.

Reply:
My understanding was this will not affecting current users' implementations using the library and so not a breaking change. I'm happy to mark it as such if this is how its maintained, but out of curiosity, could you please elaborate why?

Reply:
The `sensor_plus` and `sensor_plus_platform_interface` are in two different packages. If the signature of the method changes, but the client project doesn't pull the correct version, this leads to a compilation error.

This means that we either need to pin the exact version or do a major release.

We had this issue before with other packages.

Reply:
FYI @vbuberen is this assumption correct? Does Melos support pinning and updating the exact version of `*_platform_interface` ?

Reply:
I am not really sure here, but during our usual release routine I will check it myself that the constraints for `*_platform_interface` are updated.

Reply:
Thanks both. I've flagged it as a breaking change as @miquelbeltran suggested. Is there anything further you need me to do in regards to this?

Reply:
No, nothing else needed. Thanks